src/views/UserSettings.vue

```diff
@@ -9,6 +9,15 @@
         <v-card :title="$t('UserSettings.ChangeLanguage')" prepend-icon="mdi-earth">
           <v-divider></v-divider>
           <v-card-text>
+            <v-autocomplete
+              v-model="userSettingsForm.selectedCountry"
+              label="Country"
+              :items="countryList"
+              item-title="name"
```

Comment:
```suggestion
              item-title="native"
```
like the languages ?

Reply:
Corrected